client/src/main/kotlin/no/nav/su/se/bakover/client/oppdrag/utbetaling/UtbetalingRequestMapper.kt

```diff
@@ -77,6 +83,12 @@ internal fun toUtbetalingRequest(
                                 refDelytelseId = it.forrigeUtbetalingslinjeId?.toString(),
                                 refFagsystemId = it.forrigeUtbetalingslinjeId?.let { utbetaling.saksnummer.toString() },
                                 attestant = listOf(UtbetalingRequest.Oppdragslinje.Attestant(utbetaling.behandler.navIdent)),
+                                grad = it.uføregrad?.let { uføregrad ->
```

Comment:
Samme som over

Reply:
I motsetning til endring over, vil vel sannsynligvis denne eksistere for alle nye linjer.

Reply:
sender alltid, og banget på uføregrad :)